### PR TITLE
contrib/wdisplays: new package (1.1.1)

### PR DESCRIPTION
**Pull request by dhruv22592**

### CODE REVIEW

File: contrib/wdisplays/template.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+pkgname = "wdisplays"
+pkgver = "1.1.1"
+pkgrel = 0
+archs = ["x86_64", "aarch64", "riscv64", "ppc64le", "ppc64"]
```
Comment (nekopsykose) [inline contrib/wdisplays/template.py:4]:
```suggestion
```

File: contrib/wdisplays/template.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+build_style = "meson"
+hostmakedepends = [
+    "meson",
```
Comment (nekopsykose) [inline contrib/wdisplays/template.py:7]: sort this

File: contrib/wdisplays/template.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    "pkgconf",
+    "wayland-devel",
```
Comment (nekopsykose) [inline contrib/wdisplays/template.py:9]:
```suggestion
    "wayland-progs",
```

File: contrib/wdisplays/template.py [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    "glib-devel",
```
Comment (nekopsykose) [inline contrib/wdisplays/template.py:10]: why is this in hostmakedepends

File: contrib/wdisplays/template.py [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+pkgdesc = "GUI display configurator for wlroots compositors"
+maintainer = "dhruv22592 <dhruv22592@protonmail.com>"
+license = "GPL-3.0-or-later"
+url = "https://github.com/artizirk/wdisplays"
+source = f"{url}/archive/{pkgver}.tar.gz"
+sha256 = "2df2c56db494c8450b1c7746b0ed2af11bf637fe2838f7412146fcc5cc1a2605"
+
+
+def post_install(self):
+    self.install_license("LICENSES/GPL-3.0-or-later.txt")
```
Comment (nekopsykose) [inline contrib/wdisplays/template.py:26]: don't have to install gpl

File: contrib/wdisplays/template.py (hunk 4 above)
Comment (nekopsykose) [reply contrib/wdisplays/template.py:9]: answer: glib-compile-resources

Comment (dhruv22592) [reply contrib/wdisplays/template.py:9]: i should add glib-compile-resources?

File: contrib/wdisplays/template.py [hunk 6]
```diff
@@ -0,0 +1,25 @@
+pkgname = "wdisplays"
+pkgver = "1.1.1"
+pkgrel = 0
+build_style = "meson"
+hostmakedepends = [
+    "meson",
+    "pkgconf",
+    "wayland-progs",
```
Comment (nekopsykose) [inline contrib/wdisplays/template.py:8]: also put python-scour in hostmakedeps

File: contrib/wdisplays/template.py [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+    "glib-devel",
+]
+makedepends = [
+    "wayland-protocols",
+    "gtk+3-devel",
```
Comment (nekopsykose) [inline contrib/wdisplays/template.py:13]: sort this too

File: contrib/wdisplays/template.py (hunk 4 above)
Comment (nekopsykose) [reply contrib/wdisplays/template.py:9]: no, do nothing

**Comment by nekopsykose** [comment]
and squash the commits into one

**Comment by nekopsykose** [comment]
that's not a squash that's a merge commit :)

**Pull request closed without merging**